src/collections.rs

```diff
@@ -81,6 +76,76 @@ impl TestSuite {
     }
 }
 
+///  Builder for [`TestSuite`](struct.TestSuite.html) objects.
+#[derive(Debug, Clone, Getters)]
+pub struct TestSuiteBuilder {
+    pub name: String,
```

Comment:
Maybe builder could use a mutable instance of their coresponding class instead of duplicating all fields. What do you think?

Reply:
Changed this to only have one single field `pub testsuite: TestSuite`, but what exactly do you mean by "mutable instance"?
I mean there is no trivial way to declare an attribute as mutable, the mutability of attributes is always inherited by the holding instance depending on the context.
A "workaround" for that might be to use [std::cell::Cell](https://doc.rust-lang.org/std/cell/struct.Cell.html).

Reply:
You are right, the mutable makes no sense here. But what you implemented is exactlly what I meant 😁